### ISSUE TITLE
CLI should wait until the server is up

### ISSUE THREAD

**Issue by wilzbach** [microservices/oms#173]
The [Scala OMG template](https://github.com/microservices/scala) is currently failing:

```
> git clone https://github.com/microservices/scala
> omg run message -a name=Pete
✔ Built Docker image with name: omg/microservices/scala
✔ Started Docker container: 6e02d6720593
✔ Health check passed
✖ Failed action: `message`: RequestError: Error: read ECONNRESET
```

However, manually with Docker this works without any issues:

```
docker build -t omg-scala . 
docker run --rm -ti -p 8080:8080 omg-scala
```

and then send requests:

```
> curl -X POST -d '{"name": "pete"}' -H "Content-Type: application/json" http://localhost:8080/message
{"message":"Hello pete"}
```

I presume that the CLI is to eager when trying to send requests and maybe doesn't retry?

**Comment by Arinono** [comment]
Published a new `omg@beta` release (v0.13.0) that should fix this issue (and maybe the #174). Could you please retry using the `health` property in the `microservice.yml` following this documentation ? (https://microservice.guide/schema/health/)

**Comment by Arinono** [comment]
Tried myself (after learning how to write Scala code 😉), it seams the issue is resolved. Now the cli waits the server to return the proper health status before executing the action.
The issue with action itself should not be related to this issue.
<img width="2032" alt="Screen Shot 2019-06-21 at 2 05 00 PM" src="https://user-images.githubusercontent.com/10957531/59921444-f072b800-942d-11e9-91ed-66e91f6fe781.png">

If it is, feel free to reopen.
